src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1820,8 +1846,12 @@ void GraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(Graph &graph)
 
 //Disable ACL post-ops in fp16 to avoid performance degradation
 #if defined(OPENVINO_ARCH_ARM64)
-        if (mergedConv->getOriginalInputPrecisionAtPort(0) == ov::element::f16)
-            continue;
+        if (mergedConv->getOriginalInputPrecisionAtPort(0) == ov::element::f16) {
+            bool isFirstSupportedPostOp = mergedConv->getFusedWith().empty() && is_post_op_supported(sum);
+            if (!isFirstSupportedPostOp) {
+                continue;
+            }
+        }
 #endif
```

Comment:
Lets remove entire code block. With https://github.com/openvinotoolkit/openvino/pull/19979 introdcution it is more efficient to avoid Sum fusion at all. So lets just return false in the beginning of `FuseConvolutionSumAndConvolutionSumActivation` in case the arch is not x86

Reply:
fixed